thanks ! it starting to give good results. We still have few errors though:
- on macos: `error: Qt requires a C++11 compiler and yours does not seem to be that.`, we probably just need to activate c++11 in the test_package's CMakeList.txt
- on windows, undefined reference to `GetFileVersionInfoSizeW, GetFileVersionInfoW, VerQueryValueW, __imp_timeSetEvent , __imp_timeKillEvent , NetShareEnum, NetApiBufferFree , __imp_GetUserProfileDirectoryW `, so we probably need to add the corresponding libs to self.cpp_info.system_libs in package_info(self)
- on linux: `error: #error "You must build your code with position independent code if Qt was built with -reduce-relocations. " "Compile your code with -fPIC (and not with -fPIE)` I'm not yet sure about this one

--- a/conanfile.py
+++ b/conanfile.py
@@ -690,6 +690,13 @@ class QtConan(ConanFile):
         fu = ['include/' + f.name for f in os.scandir('include') if f.is_dir()]
         self.cpp_info.includedirs.extend(fu)
 
+        # Should it be some extra 'if' here like "if not self.options.shared" or for generator type "cmake_find_package_multi" vs "cmake"?
+        if self.settings.os == 'Windows':
+            self.cpp_info.system_libs.append('Version')   # 'Qt5Cored.lib' require 'GetFileVersionInfoW' and 'VerQueryValueW' which are in 'Version.lib' library
+            self.cpp_info.system_libs.append('Winmm')     # 'Qt5Cored.lib' require '__imp_timeSetEvent' which is in 'Winmm.lib' library
+            self.cpp_info.system_libs.append('Netapi32')  # 'Qt5Cored.lib' require 'NetApiBufferFree' which is in 'Netapi32.lib' library
+
+
     @staticmethod
     def _remove_duplicate(l):
         seen = set()

--- a/conanfile.py
+++ b/conanfile.py
@@ -695,6 +695,7 @@ class QtConan(ConanFile):
             self.cpp_info.system_libs.append('Version')   # 'Qt5Cored.lib' require 'GetFileVersionInfoW' and 'VerQueryValueW' which are in 'Version.lib' library
             self.cpp_info.system_libs.append('Winmm')     # 'Qt5Cored.lib' require '__imp_timeSetEvent' which is in 'Winmm.lib' library
             self.cpp_info.system_libs.append('Netapi32')  # 'Qt5Cored.lib' require 'NetApiBufferFree' which is in 'Netapi32.lib' library
+            self.cpp_info.system_libs.append('UserEnv')   # 'Qt5Cored.lib' require '__imp_GetUserProfileDirectoryW ' which is in 'UserEnv.Lib' library
 
 
     @staticmethod

--- a/test_package/CMakeLists.txt
+++ b/test_package/CMakeLists.txt
@@ -1,4 +1,7 @@
 cmake_minimum_required(VERSION 3.1.0)
+
+set(CMAKE_CXX_STANDARD 11)
+
 project(test_package)
 
 include(${CMAKE_BINARY_DIR}/../conanbuildinfo.cmake)

